dockerfiles/compile_extension/build-dd-trace-php

```diff
@@ -35,7 +35,7 @@ apk add --no-cache \
     libexecinfo-dev \
     make \
 
-make all CFLAGS="-std=gnu11 -O2 -g -Wall -Wextra -Werror" ECHO_ARG="-e"
+make -j all CFLAGS="-std=gnu11 -O2 -g -Wall -Wextra -Werror" ECHO_ARG="-e"
```

Comment:
I expected this to be used in the "Build extension" step, but it looks like it's not ?

Reply:
This is just used on alpine builds.